verifiers/internal/gcb/provenance.go

```diff
@@ -0,0 +1,374 @@
+package gcb
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	intoto "github.com/in-toto/in-toto-golang/in_toto"
+	slsa01 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.1"
+	dsselib "github.com/secure-systems-lab/go-securesystemslib/dsse"
+
+	serrors "github.com/slsa-framework/slsa-verifier/errors"
+	"github.com/slsa-framework/slsa-verifier/options"
+	"github.com/slsa-framework/slsa-verifier/verifiers/internal/gcb/keys"
+)
+
+var GCBBuilderIDs = []string{"https://cloudbuild.googleapis.com/GoogleHostedWorker@v0.2"}
+
+type v01IntotoStatement struct {
+	intoto.StatementHeader
+	Predicate slsa01.ProvenancePredicate `json:"predicate"`
+}
+
+type provenance struct {
+	Build struct {
+		// TODO: compare to verified provenance.
```

Comment:
Create an issue for this TODO?

Maybe comment out the entire `Build` field if it's empty?

Reply:
done in https://github.com/slsa-framework/slsa-verifier/issues/221. Thanks